src/messageComposer/middleware/textComposer/commands.ts

```diff
@@ -142,29 +148,49 @@ export const createCommandsMiddleware = (
           return next(newState);
         }
 
+        const query = triggerWithToken.slice(1);
+
+        const searchQuery = getFirstWordFromText(query);
```

Comment:
Why is it necessary to get first word from query if the query itself is just a trigger followed by chars that are not white space?

Reply:
The problem is the triggerWithToken returned null for any text except `/command`, i.e, it would also return null for text having a space and then we cannot search anything as it early exits.

Reply:
Now, if you allow spaces, the user can paste anything let's say `/giphy good morning,` in that case,e the `query` will be `giphy good morning` but the search source cannot return anything as the pattern didn't match. That is why we get the first word from the query just for the command search.